chart/epinio/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
   epinio-ui:
     registry: ghcr.io/
     repository: epinio/epinio-ui
-    tag: v1.8.1-0.0.1
+    tag: ghcr.io/epinio/epinio-ui:v1.8.1-0.0.1
```

Comment:
Bad bump, adding registry and repo into tag value.

Reply:
Fixed here: https://github.com/epinio/helm-charts/pull/444